### DISCUSSION TITLE
Regarding certificate

### DISCUSSION THREAD

**Discussion by mehboobpatel** [github/india#826] [General]
Hello official i have received the certificate of internship completion would like to know how can i produce this for college final semester internship as this certificate is not having any credentials or qr code to validate and verify, also the issued date is of 31st march and my semester of internship will be in december month of this year requesting you on behalf of other students as well to please provide a credentials/qrcode to this certificate.
so that we can upload on linkedin as well 
Please answer
@ey-mentor1 @ey-mentor2 @EY-Mentor3 @EY-Mentor4

**Comment by ey-mentor1** [comment]
Hello,
You can download the certificate and upload it to LinkedIn.
Date in certificate will be same as when your project is evaluated as 'Approved' by the mentors.
For further queries, please drop mail to FRT support team.
Thank you

**Reply by mehboobpatel** [reply]
also on the certificate which is eligible for credits its not mentioned that it is eligible for credits so how can show my college that it its eligible for credits
